code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -718,6 +727,11 @@ pub mod pallet {
 			let mut rewards_pool =
 				RewardPools::<T>::try_get(pool_id).map_err(|_| Error::<T>::RewardsPoolNotFound)?;
 
+			ensure!(
```

Comment:
Need a unit test for this?

Reply:
e6c847f51cfa3732e35821e947d7174149751f3f